DRODLib/Character.h

```diff
@@ -118,6 +118,7 @@ class CCharacter : public CPlayerDouble
 	virtual UINT   GetIdentity() const {return this->wIdentity;}
 	virtual UINT   GetResolvedIdentity() const;
 	UINT           GetNextSpeechID();
+	bool           HasBehavior(ScriptFlag::Behavior behavior) const { return behaviorFlags.count(behavior); };
```

Comment:
Suggest keeping the explicit boolean comparison `== 1` here to avoid compiler warnings in some settings regarding the implicit type casting of an integer to a bool.